src/frontend/src/lib/components/tokens/TokensSignedIn.svelte

```diff
@@ -41,9 +42,12 @@
 			handleAnimationEnd();
 		}
 	}, 250);
+
+	let loading: boolean;
+	$: $erc20UserTokensNotInitialized, (loading = $erc20UserTokensNotInitialized);
```

Comment:
Can be simplified to `$: loading = $erc20UserTokensNotInitialized;`

Reply:
corrected